OpenEdXMobile/src/main/java/org/edx/mobile/whatsnew/WhatsNewItemModel.java

```diff
@@ -8,38 +8,21 @@
 import java.util.List;
 
 public class WhatsNewItemModel implements Parcelable {
-    public enum Platform {
```

Comment:
Why have you removed the Platform enum?

Reply:
It was auto refactored before commit by android studio. I've reverted it back and pushed